test/controllers/home_controller_test.rb

```diff
@@ -1,7 +1,38 @@
 require 'test_helper'
 
 class HomeControllerTest < ActionDispatch::IntegrationTest
-  test 'the truth' do
-    assert true
+  test 'joe redirect to my_data when seeing joe_project with data_discovery' do
+    sign_in(:joe_dd)
+    @joe_project = projects(:joe_project)
+    get "/home?project_id=#{@joe_project.id}"
+    res = @response.body
+
+    assert_redirect_to(action: my_data, )
+    assert_response :success
+    puts res
+  end
+
+  test 'joe redirect to my_data when seeing joe_project with data_discovery' do
+    sign_in(:joe_dd)
+    @joe_project = projects(:joe_project)
+    get "/home?project_id=#{@joe_project.id}"
+    res = @response.body
+
+    assert_redirect_to(action: my_data, )
+    assert_response :success
+    puts res
   end
+
+  test 'joe redirect to my_data when seeing joe_project with data_discovery' do
```

Comment:
why is the same test printed three times over?

Reply:
Test was still in the works... thats why I had `[WIP] Adding some extra tests` in the top of the description. :)